neo/Ledger/Blockchain.cs

```diff
@@ -327,7 +327,7 @@ private RelayResultReason OnNewConsensus(ConsensusPayload payload)
         {
             if (!payload.Verify(currentSnapshot)) return RelayResultReason.Invalid;
             system.Consensus?.Tell(payload);
-            RelayCache.Add(payload);
+            ConsensusRelayCache.Add(payload);
```

Comment:
Maybe we can move this call before `system.Consensus?.Tell` it could be useful for search consensus payloads in the future...